tpm2d/ml.c

```diff
@@ -21,6 +21,12 @@
  * Fraunhofer AISEC <trustme@aisec.fraunhofer.de>
  */
 
+#ifdef ANDROID
```

Comment:
Ifdefs ANDROID were removed from code base.
Plz do not reintroduce legacy stuff.

Reply:
Done